site/content/en/docs/Installation/Terraform/gke.md

```diff
@@ -87,6 +87,17 @@ Configurable parameters:
 - agones_version - the version of agones to install (default is the latest version from the [Helm repository](https://agones.dev/chart/stable))
 - machine_type - machine type for hosting game servers (default is "n1-standard-4")
 - node_count - count of game server nodes for the default node pool (default is "4")
+- network - the name of the network you want your cluster and firewall rules to reside (default is "default")
```

Comment:
This isn't customizable in the gke-local or gke-module. They both hard code the network to default.

Reply:
Oh that's a good point! I was thinking of the terraform/gke module. Will remove.

Reply:
From @chrisst 's comments, actually added them to the module definitions - so leaving this in, and adding zone.